src/obfl-specification.html

```diff
@@ -1071,6 +1112,12 @@ <h4 class="include" id="L11470">dynamic-sequence</h4>
           <dd>Specifies a translation policy.</dd>
           <dd>See the <a href="#L2462">attribute description</a> for more
             information.</dd>
+		<dt>break-before [optional]</dt>
+          <dd class="no-markup">Inserts a break before the sequence begins.</dd>
+          <dd>One of 'sheet' or 'page'.</dd>
```

Comment:
Did you forget 'auto' here?

Reply:
I did! Thanks.

I have added 'auto' here and also at another place where it was missing at the `break-before`.